eox_core/tests/integration/test_grade_integration.py

```diff
@@ -0,0 +1,175 @@
+            "client_secret": cls.data["site1_data"]["client_secret"],
+            "grant_type": "client_credentials",
+        }
+        site2_data = {
+            "client_id": cls.data["site2_data"]["client_id"],
+            "client_secret": cls.data["site2_data"]["client_secret"],
+            "grant_type": "client_credentials",
+        }
+        request_url = "{}/{}".format(
+            cls.data["site1_data"]["base_url"], "oauth2/access_token/"
+        )
+        response_site1 = requests.post(request_url, data=site1_data)
+        response_site1.raise_for_status()
+        cls.data["site1_data"]["token"] = response_site1.json()["access_token"]
+        request_url = "{}/{}".format(
+            cls.data["site2_data"]["base_url"], "oauth2/access_token/"
+        )
+        response_site2 = requests.post(request_url, data=site2_data)
+        response_site2.raise_for_status()
+        cls.data["site1_data"]["token"] = response_site1.json()["access_token"]
+        cls.data["site2_data"]["token"] = response_site2.json()["access_token"]
+
+        create_enrollment(cls.data)
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def test_read_valid_default_params(self):
+        # pylint: disable=invalid-name
+        """
+        Get grades info from a user enrolled on a course without the optional
+        fields.
+        """
+        site1_data = self.data["site1_data"]
+        data = {
+            "email": site1_data["user_email"],
+            "course_id": site1_data["course"]["id"],
+        }
+        headers = {
+            "Authorization": "Bearer {}".format(site1_data["token"]),
+            "Host": site1_data["host"],
+        }
+        request_url = "{}/{}".format(site1_data["base_url"], self.data["endpoint"])
+
+        response = requests.get(request_url, data=data, headers=headers)
+        response_content = response.json()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("earned_grade", response_content)
+
+    def test_read_detail_no_policy(self):
+        # pylint: disable=invalid-name
+        """
+        Get grades info from a user enrolled on a course. Include detailed info
+        for each graded subsection.
+        """
+        site1_data = self.data["site1_data"]
+        data = {
+            "email": site1_data["user_email"],
+            "course_id": site1_data["course"]["id"],
+            "detailed": True,
+        }
+        headers = {
+            "Authorization": "Bearer {}".format(site1_data["token"]),
+            "Host": site1_data["host"],
+        }
+        request_url = "{}/{}".format(site1_data["base_url"], self.data["endpoint"])
+
+        response = requests.get(request_url, data=data, headers=headers)
+        response_content = response.json()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("earned_grade", response_content)
+        self.assertIn("section_breakdown", response_content)
```

Comment:
Maybe we should assert that the response does not contains the grade_policy string. The same applies for the other tests

Reply:
You're right, I'll change that.